src/coreclr/jit/regMaskTPOps.cpp

```diff
@@ -5,15 +5,130 @@
 
 struct regMaskTP;
 
+// ----------------------------------------------------------
+//  AddRegNumForType: Adds `reg` to the mask.
+//
+void regMaskTP::AddRegNumInMask(regNumber reg)
+{
+    SingleTypeRegSet value = genSingleTypeRegMask(reg);
+#ifdef HAS_MORE_THAN_64_REGISTERS
+    if (reg < 64)
+    {
+        low |= value;
+    }
+    else
+    {
+        high |= value;
+    }
```

Comment:
Would it be more efficient to inline the computation of `value` in the "else" case as `reg - 64`?

Reply:
The `mask` value for registers > 63, will anyway start from `0x0`, so it should be fine, the way we are doing currently.